### ISSUE TITLE
Rocket.Chat (self-hosted Slack alternative)

### ISSUE THREAD

**Issue by clonm** [cloyne/network#117]
Mitar recommends [Rocket.Chat](https://rocket.chat/) over [Riot.im](https://about.riot.im/)/[Matrix](https://matrix.org/blog/home/). Could also be part of [migrating to NextCloud](https://github.com/cloyne/docker-owncloud/issues/1).

**Comment by kngnm** [comment]
An Update:

- There is an official docker image for rocket chat server (docker pull rocket.chat).   I have not figured out how to give the image persistent memory.  I have experimented with using docker volumes, but they seem to cause the image to crash.  
- There is also a REST API for rocketchat.  I was able to use this to write a script to automatically add channels and users from the house list spread sheet.
- I have a raspberry pi with a nginx reverse proxy server installed on it.  If it could be set up to have ip address 64.62.133.46 on the cloyne network (presumably by attaching it to the router and configuring it like one of Cloyne's servers)  I could host the rocket chat server by ssh tunneling into the pi from the host machine.  I think this solution may be less time intensive than routing the packets.  I would only need help setting up the pi on the cloyne network; the rest I have already prepared at home.  Let me know if you see any flaws with this solution and/or would be willing to help me implement it. 

@clonm
